pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.poetry]
+name = "ocs-archive"
+version = "0.2.10"
```

Comment:
The setup.py deploy to pypi used the tagged version, but now you are explicitly setting the version here. Does this mean we have to be careful to keep this version in sync with the tagged version so it is deployed to pypi as the correct version now?

Reply:
Thought about this too - maybe we can do something like [this](https://nadeauinnovations.com/post/2020/08/one-version-to-rule-them-all-keeping-your-python-package-version-number-in-sync-with-git-and-poetry/)?

We'd have a dummy placeholder in the `pyproject.toml`, but then on tag we'd update it when pushing to PyPi. The downside of this is when the user installs it locally from the git repo, it would show them as having the dummy version installed.

Reply:
IMO, it's semantically more correct to keep the version in `pyproject.toml` file and then commit & tag. Tags should be thought of as pointers to the complete source code w/ the versions included. To help avoid synchronization errors, I would create a manually trigger-able workflow that does that for us, e.g. sourcing it from `poetry version -s`.